src/app/components/courses/courses.component.ts

```diff
@@ -65,9 +97,15 @@ export class CoursesComponent implements OnInit {
     this.currentPage++;
     const count = (this.currentPage * this.COURSES_PER_PAGE).toString();
 
-    this._coursesService.getCourses({start: '0', count})
+    const subscription = this._coursesService.getCourses({start: '0', count})
```

Comment:
Better to give concrete names  e.g. getCoursesSubscription  (you can have a lot of subscriptions in one component)